plugins/catalog-backend-module-gitlab/package.json

```diff
@@ -43,12 +44,14 @@
     "lodash": "^4.17.21",
     "msw": "^0.42.0",
     "node-fetch": "^2.6.7",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "uuid": "^8.0.0"
   },
   "devDependencies": {
-    "@backstage/backend-test-utils": "^0.1.25-next.2",
-    "@backstage/cli": "^0.17.2-next.2",
-    "@types/lodash": "^4.14.151"
+    "@backstage/backend-test-utils": "^0.1.25-next.1",
+    "@backstage/cli": "^0.17.2-next.1",
+    "@types/lodash": "^4.14.151",
+    "@types/uuid": "^8.0.0"
```

Comment:
Since we've made a release, you will want to update these versions now (so that the yarn.lock file below effectively loses all of its changes). There's a helper script `node scripts/verify-local-dependencies.js` that might help with this.